rule.php

```diff
@@ -0,0 +1,157 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * Implementaton of the quizaccess_seb plugin.
+ *
+ * @package    quizaccess_seb
+ * @author     Andrew Madden <andrewmadden@catalyst-au.net>
+ * @copyright  2019 Catalyst IT
+ * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+defined('MOODLE_INTERNAL') || die();
+
+require_once($CFG->dirroot . '/mod/quiz/accessrule/accessrulebase.php');
+
+class quizaccess_seb extends quiz_access_rule_base {
+
+    /**
+     * quizaccess_seb constructor.
+     */
+    public function __construct($quizobj, $timenow) {
+        parent::__construct($quizobj, $timenow);
+    }
+
+    /**
+     * @param quiz $quizobj
```

Comment:
Missing doc?

Reply:
Fixed

Reply:
As we are aiming to push this code to core, let's start to follow strict standards from very beginning. We need to have description for all parameters in all methods.